Sources/DeckUI/Views/SlideNavigation.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  SlideNavigation.swift
+//  
+//
+//  Created by Zachary Brass on 3/18/23.
+//
+
+import SwiftUI
+
+public struct SlideNavigationToolbarButtons: View {
+    public var body: some View {
+        Group {
+            Button {
+                withAnimation {
+                    PresentationState.shared.previousSlide()
+                }
+            } label: {
+                Label("Previous", systemImage: "arrow.left")
+            }.keyboardShortcut(.leftArrow, modifiers: [])
+            
+            Button {
+                withAnimation {
+                    PresentationState.shared.nextSlide()
+                }
+            } label: {
+                Label("Next", systemImage: "arrow.right")
+            }.keyboardShortcut(.rightArrow, modifiers: [])
+            
+            Button {
+                NotificationCenter.default.post(name: .keyDown, object: nil)
+            } label: {
+                Label("Down", systemImage: "arrow.down")
+            }.keyboardShortcut(.downArrow, modifiers: [])
+            
+            Button {
+                NotificationCenter.default.post(name: .keyUp, object: nil)
+            } label: {
+                Label("Up", systemImage: "arrow.up")
+            }.keyboardShortcut(.upArrow, modifiers: [])
+        }
+    }
+    public init() {
+        
+    }
```

Comment:
Can probably remove this init!

Reply:
I put this in there to change the init method from internal to public